recipes-app/node-red-gpio/files/0005-mraa-gpio-dout-Add-D14-D19-for-IOT2050.patch

```diff
@@ -0,0 +1,72 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Baocheng Su <baocheng.su@siemens.com>
+Date: Thu, 28 Dec 2023 10:19:31 +0800
+Subject: [PATCH] mraa-gpio: dout: Add D14-D19 for IOT2050
+
+This was lost during last upstreaming.
```

Comment:
Did we have multiple upstreamings, or why "last"?

Reply:
Ack, and will reword.